### PR TITLE
Feature/admin abilities

### PR DESCRIPTION
**Pull request by TylerRockwell**
### Why?
https://github.com/smashingboxes/apprenticeship/tree/master/Curriculum/Backend/bookstore
Admins need access to all orders to do their jobs properly

### What Changed?
Added Order controller to Admin namespace with index action
Added views and tests for said action

### CODE REVIEW

File: features/step_definitions/administration_steps.rb [hunk 1]
```diff
@@ -31,6 +31,10 @@
   FactoryGirl.create(:book, title: book_title, price: price)
 end
 
+Given(/^There are orders$/) do
+  @orders = FactoryGirl.create_list(:order, 20)
```
Comment (eklatz) [inline features/step_definitions/administration_steps.rb]: Nice use of `create_list`

Comment (albaer) [inline features/step_definitions/administration_steps.rb]: Nice use of `create_list`.

File: config/routes.rb [hunk 2]
```diff
@@ -10,6 +10,7 @@
   namespace :admin do
     resources :books
     resources :dashboard, only: :index
```
Comment (albaer) [inline config/routes.rb]: I realize this isn't part of this pr, but is dashboard actually a resource? If it's just a page you can just do `get 'dashboard'` and whatever options you need to pass. If it's a singular resource you can use `resource 'dashboard', only: :show`.

**Comment by albaer** [comment]
Nice work! :+1:

File: config/routes.rb (hunk 2 above)
Comment (TylerRockwell) [inline config/routes.rb]: It's just a page. I changed it. Thanks for pointing this out.

**Comment by TylerRockwell** [comment]
Thanks, @albaer !

**Pull request merged**